index.d.ts

```diff
@@ -64,9 +64,18 @@ declare const matcher: {
 
 	matcher.isMatch('UNICORN', 'unicorn', {caseSensitive: true});
 	//=> false
+
+	matcher.isMatch(['foo', 'bar'], 'f*');
+	//=> true
+
+	matcher.isMatch(['foo', 'bar'], ['a*', 'b*']);
+	//=> true
+
+	matcher.isMatch('unicorn', ['tri*', 'UNI*'], {caseSensitive: true});
+	//=> false
```

Comment:
This change needs to be reflected in the readme too.

Reply:
Done, added the array form to the docs and examples.